hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/RedisBackendResolver.java

```diff
@@ -43,6 +50,14 @@
     setAttributeIfExist(event, entityBuilder, RawSpanConstants.getValue(Redis.REDIS_ARGS));
     setAttributeIfExist(
         event, entityBuilder, OTelDbSemanticConventions.DB_CONNECTION_STRING.getValue());
-    return Optional.of(entityBuilder.build());
+
+    setAttributeForFirstExistingKey(
```

Comment:
If the attribute was not set in the entity before, let's not add it to the entity

Reply:
we don't have to set db operation with an entity.